emoji-google-compat/src/main/java/com/vanniktech/emoji/googlecompat/GoogleCompatEmojiDrawable.java

```diff
@@ -0,0 +1,64 @@
+package com.vanniktech.emoji.googlecompat;
+
+import android.graphics.Canvas;
+import android.graphics.ColorFilter;
+import android.graphics.Paint;
+import android.graphics.PixelFormat;
+import android.graphics.Rect;
+import android.graphics.drawable.Drawable;
+import android.support.text.emoji.EmojiCompat;
+import android.support.text.emoji.EmojiSpan;
+import android.text.Spanned;
+import android.text.TextPaint;
+
+/**
+ * An emoji drawable backed by a span generated by the Google emoji support library.
+ */
+public class GoogleCompatEmojiDrawable extends Drawable {
```

Comment:
final & package private so others can't use it and we won't have to support it in the public API